sockeye/arguments.py

```diff
@@ -214,7 +241,30 @@ def add_model_parameters(params):
                               default=False,
                               help="Add residual connections to stacked RNNs if --rnn-num-layers > 3. "
                                    "(see Wu ETAL'16). Default: %(default)s.")
+    model_params.add_argument('--rnn-context-gating', action="store_true",
+                              help="Enables a context gate which adaptively weighs the RNN decoder input against the"
+                                   "source context vector before each update of the decoder hidden state.")
+
+    # transformer arguments
+    model_params.add_argument('--transformer-model-size',
```

Comment:
that is true. I thought more regarding consistence. Anyway, no strong feelings about this ;)